src/ui/src/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const API_URL = process.env.NEXT_PUBLIC_API_URL || "/api";
```

Comment:
@johanna-skylight any reason to not just make the API_URL an environment variable?  Is `NEXT_PUBLIC_API_URL` needed for something else?

Reply:
`NEXT_PUBLIC_API_URL` is the environment variable and is being defined inline when the application starts with [the dev script](https://github.com/CDCgov/RecordLinker/blob/4799c8643cbccf251e3f86128e3bf81b5f24c8d0/src/ui/package.json#L6C4-L6C58) for dev mode. The prefix `NEXT_PUBLIC` is necessary.

 The constant `API_URL` is just a way to not duplicate the logic that figures the api prefix. It needs to be set as  /api for prod builds or when the app runs without ui in local so that is the fallback value,  but in dev mode this needs to be set to `localhost:8080`.